vendor/engines/sanger_sequencing/app/controllers/sanger_sequencing/admin/batches_controller.rb

```diff
@@ -5,14 +5,30 @@ module Admin
     class BatchesController < BaseController
 
       admin_tab :all
-      # layout "two_column"
-      # before_filter { @active_tab = "admin_sanger_sequencing" }
+      layout "two_column", only: [:index]
+      before_filter(only: :index) { @active_tab = "admin_sanger_sequencing" }
+
+      def index
+        @batches = Batch.order(created_at: :desc).paginate(page: params[:page])
+      end
 
       def new
```

Comment:
FYI, I'll also have authorization around the batches controller in the next PR, as well as locking the batch to a specific facility.